differential/customfield/DifferentialBugzillaBugIDValidator.php

```diff
@@ -13,6 +13,17 @@ public static function validate($bug_id, $account_phid) {
 
     $bug_id = self::formatBugID($bug_id);
 
+    // Check to see if the user is an admin; if so, don't validate bug existence
+    // because the admin account may not have a BMO account ID associated with it
+    $users = id(new PhabricatorPeopleQuery())
+      ->setViewer(PhabricatorUser::getOmnipotentUser())
+      ->withPHIDs(array($account_phid))
+      ->withIsAdmin(true)
+      ->execute();
```

Comment:
This is a complex-looking bit of code.  Should the `isUserAdmin()` check be in a utility function to keep this function's size down, and in case we want to reuse the check elsewhere?